backend/molgenis-emx2-rdf/src/main/java/org/molgenis/emx2/rdf/RDFService.java

```diff
@@ -393,13 +429,16 @@ public void rowsToRdf(final ModelBuilder builder, Table table, final String rowI
       if (table.getMetadata().getTableType() == TableType.ONTOLOGIES) {
         builder.add(subject, RDF.TYPE, IRI_CODED_VALUE_DATATYPE);
         builder.add(subject, RDF.TYPE, OWL.CLASS);
+        builder.add(subject, RDF.TYPE, SKOS.CONCEPT);
         builder.add(subject, RDFS.SUBCLASSOF, tableIRI);
-
+        builder.add(subject, SKOS.IN_SCHEME, tableIRI);
         if (row.getString("name") != null) {
           builder.add(subject, RDFS.LABEL, Values.literal(row.getString("name")));
+          builder.add(subject, SKOS.PREF_LABEL, Values.literal(row.getString("name")));
         }
         if (row.getString("label") != null) {
           builder.add(subject, RDFS.LABEL, Values.literal(row.getString("label")));
+          builder.add(subject, SKOS.ALT_LABEL, Values.literal(row.getString("name")));
```

Comment:
Should we make it the skos:prefLabel if name == null? And again we don't need rdfs:label if we use skos:altLabel.